### DISCUSSION TITLE
Why ceph mon use more memory because of the long time "clock skew"?

### DISCUSSION THREAD

**Discussion by zhucan** [rook/rook#11037] [Q&A]
41686 167       20   0   10.8g  10.3g  16756 S  45.2 66.6   5326:51 ceph-mon                                                           

if we don't set limits of the ceph mon pod. there is "clock skew" in the ceph cluster.

**Comment by travisn** [comment]
Do you have memory limits on the mons? They will contain the memory better in that case.

**Reply by zhucan** [reply]
no, but I want to know, in what conditions that the ceph mon will use more memory.

**Reply by travisn** [reply]
Not sure, but if you need to constrain it, ceph will respect the memory limits

**Reply by zhucan** [reply]
how much is suitable for ceph mon fo the mem?

**Reply by travisn** [reply]
The minimum pod recommendations are documented [here](https://rook.io/docs/rook/latest/CRDs/Cluster/ceph-cluster-crd/#cluster-wide-resources-configuration-settings). You might try 2GB.

**Reply by zhucan** [reply]
it will impact performance of the ceph cluster,  if the mem of the ceph-mon is too less?
